language-design.md

```diff
@@ -102,7 +200,27 @@ you do so? If not, why not?
 
 **Response**
 
-
+It is possible that they are at odds because of the extent to which natural 
+language is used. In the examples given of Quorum, we saw that a for loop was
+replaced by `repeat 10 times` and `int` was replaced with `integer`. These are
+pretty limited changes, and it seems that it is simply a replacement of keywords
+to clarify meaning. However, if a programming language that was talked about by
+William Cook was able to string together long chains of "keywords" such that
+the code did resemble natural language sentences, then it could be extremely
+misleading what is actually part of the language. 
+
```

Comment:
It seems like the distinction you are making is between incorporating natural language in the vocabulary versus the grammar.  I agree that having natural language as part of the grammar makes writing code much more difficult.  What is your opinion about how much natural language to use in the vocabulary?  Is it worth it to write `repeat` instead of `for`, or maybe `integer` instead of `int` hundreds of times for the marginal clarity that it provides?  Do you think it depends on the language?